.github/workflows/playwright

```diff
@@ -0,0 +1,15 @@
+name: Playwright Setup
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install dependencies for all browsers
```

Comment:
See the comment about redundant dependency installs on PR #14 .
Also the related comment about running actual tests as part of the CI script.

Reply:
From my friend chatgpt:

_Based on the code provided, there does not seem to be any redundant dependency installs.

The first step installs the dependencies for all browsers using the command npx playwright install-deps.

The second step installs the dependencies for Chromium only using the command npx playwright install-deps chromium. This is necessary because not all browsers require the same dependencies, and installing dependencies for all browsers may result in unnecessary downloads and installation of packages.

Therefore, each step is installing the necessary dependencies for the intended browser and there does not seem to be any redundant installations_.

Reply:
Can you please elaborate on this? "Also the related comment about running actual tests as part of the CI script."

I don't follow.